infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -126,28 +72,41 @@ public void delete(@Nonnull ConflictTxStream key, Long globalAddress, @Nonnull R
      * @return global address
      */
     public Long getIfPresent(ConflictTxStream conflictKey) {
-        return conflictCache.getIfPresent(conflictKey);
+        return cacheHashtable.get(conflictKey);
+    }
+
+    private long firstAddress() {
+        if (cacheEntries.peek() == null)
+            return Address.NOT_FOUND;
+        return cacheEntries.peek().txVersion;
     }
 
     /**
-     * Invalidate all records up to a trim mark.
+     * Invalidate one record.
```

Comment:
It would be better if the JavaDoc is more descriptive. Does this function invalidate the first record?